docs/book/src/cronjob-tutorial/testdata/project/Makefile

```diff
@@ -63,7 +63,7 @@ generate: controller-gen
 
 # Build the docker image
 docker-build: test
-	docker build . -t ${IMG}
+	docker build -t ${IMG} .
```

Comment:
This belongs to `v2`, so please discard this change.

Reply:
Done with cced240f -> ccb11276.